package.json

```diff
@@ -53,6 +54,7 @@
     "react-dnd": "^11.1.3",
     "react-dnd-html5-backend": "^11.1.3",
     "react-dom": "^16.13.1",
+    "react-helmet": "^6.1.0",
```

Comment:
[note] This is used to preload various assets I believe.

Reply:
Though I wonder if there is a simpler way we can do it without needing to bring in a whole new dependency?

Reply:
Helmet library added for the specific preloading reason in this PR, but in general it allows React components to add tags to the `<head>` part of the document.

Reply:
Yup, i'm aware of the library and its usages. I'm just not sure that we need it. I expect we can do the preload stuff in a smaller/simpler way using hooks + JS APIs or similar.

Reply:
I mentioned the usage for completeness in this discussion. Still, if there's a way to escape  the irony of adding library to reduce bundle size, I'm in.

> I expect we can do the preload stuff in a smaller/simpler way using hooks + JS APIs or similar.

I can look into that, had noticed a `setTitle` hook being used, so I guess it should be similar.

Reply:
Using hook instead of `react-helmet` in e9da1d5788e915f6de13a7aa0315601017c75b73. 

Code is simplified, more in line with the rest app code and no extra library included i.e. no extra KBs in bundles.

Reply:
Nice!